src/preact.d.ts

```diff
@@ -88,6 +88,8 @@ declare namespace JSX {
 	}
 
 	interface SVGAttributes {
+		class?:string | { [key:string]: boolean };
+		className?:string | { [key:string]: boolean };
```

Comment:
Not sure if I'm missing something, but is there a reason why `className` can accept an object?